src/coreclr/vm/codeman.cpp

```diff
@@ -2676,40 +2677,49 @@ CodeHeader* EEJitManager::allocCode(MethodDesc* pMD, size_t blockSize, size_t re
 
         _ASSERTE(IS_ALIGNED(pCode, alignment));
 
-        // Initialize the CodeHeader *BEFORE* we publish this code range via the nibble
-        // map so that we don't have to harden readers against uninitialized data.
-        // However because we hold the lock, this initialization should be fast and cheap!
-
         pCodeHdr = ((CodeHeader *)pCode) - 1;
 
+        *pAllocatedSize = sizeof(CodeHeader) + totalSize + reserveForJumpStubs;
```

Comment:
I do not think `reserveForJumpStubs` should be included in this number.

Reply:
Removed